Tests/XCUITests/ActivityStreamTest.swift

```diff
@@ -11,32 +11,21 @@ let allDefaultTopSites = ["Facebook", "YouTube", "Amazon", "Wikipedia", "Twitter
 class ActivityStreamTest: BaseTestCase {
     let TopSiteCellgroup = XCUIApplication().cells[AccessibilityIdentifiers.FirefoxHomepage.TopSites.itemCell]
 
-    let testWithDB = ["testActivityStreamPages", "testTopSites2Add"]
+    let testWithDB = ["testTopSites2Add", "testTopSitesRemoveAllExceptDefaultClearPrivateData"]
 
     // Using the DDDBBs created for these tests containing enough entries for the tests that used them listed above
-    let pagesVisitediPad = "browserActivityStreamPagesiPad.db"
-    let pagesVisitediPhone = "browserActivityStreamPagesiPhone.db"
+    let pagesVisited = "browserActivityStreamPages-places.db"
 
     override func setUp() {
         // Test name looks like: "[Class testFunc]", parse out the function name
         let parts = name.replacingOccurrences(of: "]", with: "").split(separator: " ")
         let key = String(parts[1])
         if testWithDB.contains(key) {
-            // for the current test name, add the db fixture used
-            if iPad() {
-                launchArguments = [LaunchArguments.SkipIntro,
-                                   LaunchArguments.SkipWhatsNew,
-                                   LaunchArguments.SkipETPCoverSheet,
-                                   LaunchArguments.LoadDatabasePrefix + pagesVisitediPad,
-                                   LaunchArguments.SkipContextualHints]
-            } else {
-                launchArguments = [LaunchArguments.SkipIntro,
-                                   LaunchArguments.SkipWhatsNew,
-                                   LaunchArguments.SkipETPCoverSheet,
-                                   LaunchArguments.LoadDatabasePrefix + pagesVisitediPhone,
-                                   LaunchArguments.SkipContextualHints,
-                                   LaunchArguments.TurnOffTabGroupsInUserPreferences]
-            }
+            launchArguments = [LaunchArguments.SkipIntro,
```

Comment:
We can use the same database for both iPhone and iPad. The database consists of 12 recently visited page. On iPhone, the first 8 is shown. On iPad, all 12 is shown.